src/vmm/src/builder.rs

```diff
@@ -1294,182 +1262,78 @@ pub mod tests {
     #[test]
     fn test_error_messages() {
         use builder::StartMicrovmError::*;
-        let err = CreateBlockDevice(io::Error::from_raw_os_error(0));
-        assert_eq!(
-            format!("{}", err),
-            format!(
-                "Unable to seek the block device backing file due to invalid permissions or \
-                 the file was deleted/corrupted. Error number: {}",
-                io::Error::from_raw_os_error(0)
-            )
-        );
+        let err = AttachBlockDevice(io::Error::from_raw_os_error(0));
+        let _ = format!("{}{:?}", err, err);
```

Comment:
What is the point of calling format here without checking anything?

Reply:
It tests that `Display` and `Debug` are implemented.

Reply:
Ok. It looks a bit like it is not needed to call that, but that's just a nit.